include/ck_tile/ops/gemm/pipeline/gemm_pipeline_ag_bg_cr_comp_v5.hpp

```diff
@@ -106,237 +143,486 @@ struct GemmPipelineAgBgCrCompV5 : public BaseGemmPipelineAgBgCrCompV5<Problem>
+                                            .at(I0))>;
+            using bcopy_dram_type =
+                remove_cvref_t<decltype(PipelineImplBase::GetBWindows(b_dram_block_window_tmp,
+                                                                      b_lds_block,
+                                                                      b_lds_load_tile_distr)
+                                            .at(I0))>;
+
+            using a_copy_lds_window_type =
+                remove_cvref_t<decltype(PipelineImplBase::GetAWindows(a_dram_block_window_tmp,
+                                                                      a_lds_block,
+                                                                      a_lds_load_tile_distr)
+                                            .at(I1))>;
+            using b_copy_lds_window_type =
+                remove_cvref_t<decltype(PipelineImplBase::GetBWindows(b_dram_block_window_tmp,
+                                                                      b_lds_block,
+                                                                      b_lds_load_tile_distr)
+                                            .at(I1))>;
+
+            using a_lds_load_tile_distr_type =
+                remove_cvref_t<decltype(PipelineImplBase::GetAWindows(a_dram_block_window_tmp,
+                                                                      a_lds_block,
+                                                                      a_lds_load_tile_distr)
+                                            .at(I2))>;
+            using b_lds_load_tile_distr_type =
+                remove_cvref_t<decltype(PipelineImplBase::GetBWindows(b_dram_block_window_tmp,
+                                                                      b_lds_block,
+                                                                      b_lds_load_tile_distr)
+                                            .at(I2))>;
+
+            auto&& aWindows = PipelineImplBase::GetAWindows(
+                a_dram_block_window_tmp, a_lds_block, a_lds_load_tile_distr);
+            auto&& bWindows = PipelineImplBase::GetBWindows(
+                b_dram_block_window_tmp, b_lds_block, b_lds_load_tile_distr);
+
+            // A DRAM tile window for load
+            // A LDS tile window for store
+            // A LDS tile for block GEMM
+            acopy_dram_type& a_copy_dram_window           = aWindows.at(I0);
+            a_copy_lds_window_type& a_copy_lds_window     = aWindows.at(I1);
+            a_lds_load_tile_distr_type& a_lds_gemm_window = aWindows.at(I2);
+
+            // B DRAM tile window for load
+            // B LDS tile window for store
+            // B LDS tile for block GEMM
+            bcopy_dram_type& b_copy_dram_window           = bWindows.at(I0);
+            b_copy_lds_window_type& b_copy_lds_window     = bWindows.at(I1);
+            b_lds_load_tile_distr_type& b_lds_gemm_window = bWindows.at(I2);
+
```

Comment:
We can change all these acopy_dram_type,a_copy_lds_window_type,a_lds_load_tile_distr_type... to the auto since we get this from tuple